python/tskit/genotypes.py

```diff
@@ -222,6 +223,27 @@ def copy(self) -> Variant:
         variant_copy._ll_variant = self._ll_variant.restricted_copy()
         return variant_copy
 
+    def __str__(self) -> str:
+        """
+        Return a plain text summary of the contents of a variant.
+        """
+        variant_rows = [
+            ["Site id", str(self.site.id)],
+            ["Samples", str(len(self.samples))],
+            ["Genotypes", str(len(self.genotypes))],
+            ["Alleles", str(self.num_alleles)],
```

Comment:
Would be nice to give the allele counts here.

Reply:
Would it be nice to implement https://github.com/tskit-dev/tskit/issues/2390 to help with this?